src/integration_tests/integration_test.go

```diff
@@ -14,19 +13,20 @@ func init() {
 		log.Fatalf("Failed to decode uptime service configuration: %v", err)
 	}
 
-	// Setup AWS Keyspaces database
 	setAppConfig("all")
 	config := getAppConfig()
```

Comment:
thought: if this is not part of keyspaces, perhaps it should be moved above `// Setup AWS Keyspaces database`?

Reply:
Indeed it is above:
```
	setAppConfig("all")
	config := getAppConfig()

	// Setup AWS Keyspaces database
       ...
```

Reply:
:facepalm: sorry for the noise